src/radical/entk/appman/config.json

```diff
@@ -7,7 +7,7 @@
     "reattempts"      : 3,
     "resubmit_failed" : false,
     "autoterminate"   : true,
-    "write_workflow"  : false,
+    "write_workflow"  : true,
```

Comment:
Why was that changed?  I mean, I don't mind, just wonder about the reason...

Reply:
No reason, I set up to test things and I think it just got committed. I will revert it. There is no reason to write a file every time EnTK is run.